commands/config_test.go

```diff
@@ -1,104 +0,0 @@
-package commands
```

Comment:
I felt that way before, but now I actually think the opposite is true.  It's very hard to tell just from looking at the filesystem which parts of the code have tests and which need them.  We haven't had any "Hey, noticed there was an empty test file so I submitted a patch" type contributions, so I think we just have to be stricter in code review.